src/Plugin/Field/FieldFormatter/InstagramEmbedFormatter.php

```diff
@@ -34,10 +38,15 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
     foreach ($items as $delta => $item) {
       $matches = [];
       foreach (Instagram::$validationRegexp as $pattern => $key) {
-        if (preg_match($pattern, $item->value, $matches)) {
-          break;
```

Comment:
We can break after the first match is found since we use just first item anyway (below).